monitor.py

```diff
@@ -529,7 +529,15 @@ def parse_entry(self, line: bytes) -> MutableMapping[str, Any]:  # noqa: C901, C
                 # Odyssey: bool
                 self.cmdr = entry['Commander']
                 # 'Open', 'Solo', 'Group', or None for CQC (and Training - but no LoadGame event)
-                self.mode = entry.get('GameMode')
+                if not entry.get('Ship') and not entry.get('GameMode') or entry.get('GameMode').lower() == 'cqc':
```

Comment:
Just noticed this, if in `entry` key `GameMode` doesn't exists then `entry.get('GameMode').lower()` will raise `AttributeError: 'NoneType' object has no attribute 'lower'`, going to fix it by adding default value `''`.